src/components/ai/ModelSelector.module.css

```
@@ -319,20 +319,27 @@
 
+.infoInline:hover {
+	color: var(--text-primary);
+	border-color: color-mix(in srgb, var(--border-default) 75%, transparent);
+	background: color-mix(in srgb, var(--bg-secondary) 85%, transparent);
+}
+
 .infoInline svg {
 	display: block;
 	width: 14px;
 	height: 14px;
 	color: currentColor;
```

Stylelint (17.5.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)